tests/sdk/README.md

```diff
@@ -0,0 +1,22 @@
+# Integration Tests
+
+This directory contains integration tests for the sdk.  These run against the Recall network defined by the following env vars:
+  - `RECALL_PRIVATE_KEY`, a private key for a wallet that has funds on the parent chain, RECALL, and credits
+  - `RECALL_NETWORK`, one of `localnet` or `testnet`. (eventually `mainnet` may be supported)
+  - `RECALL_AUTH_TOKEN`, Optional evm subnet auth token, if needed for the given target network.
+
+An example of running these tests against localnet Anvil default account 8 follows:
+`RECALL_NETWORK=localnet RECALL_PRIVATE_KEY=0xdbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97 cargo test --test '*' -- --nocapture --ignored`
+
+## Anvil Test Accounts and Private Keys
+
+- 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 (0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80)
```

Comment:
Just out of curiosity: where is this info required?

Reply:
These can be used to specify a particular `RECALL_PRIVATE_KEY` for the tests.